### PR TITLE
docs: add AssignmentSheet description & Constraints

### PR DESCRIPTION
**Pull request by HrRn**
## Overview
I add a description and constraints of 'AssignmentSheet'

## Changes

Group 1

## Assignee Checklist:

- [x] The base branch is correct (no accidental merges)

## Reviewer Checklist

- Code readability and simplicity
- Follows best practices and coding standards
- Understandable and maintainable code

### CODE REVIEW

File: docs/docs/modeling.md [hunk 1]
```diff
@@ -114,6 +114,25 @@ classDiagram
...
+
+The start date of the task to be performed by Tenant.
+
+Constraints：
+
+- Owner must select startDate.
+
+### endDate:
+
+The end day the task must be completed.
```
Comment (akiomatic) [inline docs/docs/modeling.md:130]: You might want to mention the timezone here.

File: docs/docs/modeling.md [hunk 2]
```diff
@@ -114,6 +114,25 @@ classDiagram
 
 ## AssignmentSheet - 分担票
 
+Tenant's task assignment table.  
+The assignmentSheet is including startDate,endDate and TenantsWork.
```
Comment (akiomatic) [inline docs/docs/modeling.md:118]: I think it's better to say this `AssignmentSheet` cannot exist without `StartDate`, `EndDate` and `TenantsWork` here.

**Review by akiomatic — CHANGES_REQUESTED** [review]
Thank you for making the description & constraints for `AssignmentSheet`.
I would appreciate it if you could read my comments.

File: docs/docs/modeling.md (hunk 2 above)
Comment (HrRn) [reply docs/docs/modeling.md:118]: Thanks for the advice. I have added the constraint.

File: docs/docs/modeling.md [hunk 3]
```diff
@@ -114,6 +114,34 @@ classDiagram
...
+The start date of the task to be performed by Tenant.
+
+Constraints：
+
+- Owner must select startDate.
+- The time zone is PST (Pacific Standard Time).
+
+### endDate:
+
+The end day the task must be completed.
```
Comment (yuki-92c) [inline docs/docs/modeling.md:135]: Thanks for the work, in my opinion, "The deadline for completing the task." is easier to grasp

**Review by akiomatic — APPROVED** [review]
LGTM!
I apologize for the late review.

**Review by Akikaze1119 — APPROVED** [review]
Thank you! LGTM👍

**Review by yuki-92c — APPROVED** [review]
Thanks. LGTM

**Review by KaiKoide — APPROVED** [review]
LGTM:)

**Review by akiomatic — APPROVED** [review]
LGTM!

**Pull request merged**